projects/ngx-extension/src/lib/operators/combine-latest-eager.operator.ts

```diff
@@ -34,29 +34,24 @@ export function combineLatestEager<T>(
   sources: Array<Observable<T>> | Record<string, Observable<T>>,
   startWithNullForAll = false,
 ): Observable<Array<T | null> | Record<string, T | null>> {
+  function observableMapper<T>(observable: Observable<T>) {
+    if (startWithNullForAll) {
+      return observable.pipe(startWith(null));
+    } else {
+      // Check if observable is a Subject, if true, apply startWith(null)
+      return observable instanceof Subject ? observable.pipe(startWith(null)) : observable;
+    }
+  }
+
   if (Array.isArray(sources)) {
     // If sources is an array of observables
-    return combineLatest(
-      sources.map((obs$) => {
-        if (startWithNullForAll) {
-          return obs$.pipe(startWith(null));
-        } else {
-          // Check if obs$ is a Subject, if true, apply startWith(null)
-          return obs$ instanceof Subject ? obs$.pipe(startWith(null)) : obs$;
-        }
-      }),
-    );
+    return combineLatest(sources.map(observableMapper));
   } else if (typeof sources === 'object' && sources !== null) {
     // If sources is a dictionary of observables
     const observables: Record<string, Observable<T | null>> = {};
 
-    for (const [key, value] of Object.entries(sources)) {
-      if (startWithNullForAll) {
-        observables[key] = value.pipe(startWith(null));
-      } else {
-        // Check if obs$ is a Subject, if true, apply startWith(null)
-        observables[key] = value instanceof Subject ? value.pipe(startWith(null)) : value;
-      }
+    for (const [ key, value ] of Object.entries(sources)) {
```

Comment:
Excellent change! Thanks @KingMario ! 

It seems the prettier rule is not working well. Would you please update this line without spaces between the bracket and the variable? Like `[key, value]` not `[ key, value ]`.  Thanks again!